resources/namespaces/two_namespaces_two_users/namespaces.yaml

```diff
@@ -14,11 +14,35 @@ namespaces:
           - apiGroups: [""]
             resources: ["pods"]
             verbs: ["get", "list", "watch"]
+          - apiGroups: [""]
+            resources: ["pods/log", "pods/exec", "pods/attach", "pods/portforward"]
+            verbs: ["get"]
+          - apiGroups: [""]
+            resources: ["configmaps", "secrets"]
+            verbs: ["get"]
+          - apiGroups: [""]
+            resources: ["persistentvolumeclaims"]
+            verbs: ["get", "list"]
+          - apiGroups: [""]
+            resources: ["events"]
+            verbs: ["get"]
```

Comment:
seems like these could all be set on the namespace-defaults.yaml file? that way we only specify rules in the namespaces.yaml when we want to override defaults for a particular user/namespace.